integration-cli/docker_cli_search_test.go

```diff
@@ -16,10 +16,20 @@ func (s *DockerSuite) TestSearchOnCentralRegistry(c *check.C) {
 }
 
 func (s *DockerSuite) TestSearchStarsOptionWithWrongParameter(c *check.C) {
-	out, _, err := dockerCmdWithError("search", "--stars=a", "busybox")
+	out, _, err := dockerCmdWithError("search", "--filter has-stars", "=a", "busybox")
```

Comment:
I don't think this check is necessary anymore, it's the same you wrote in line 28.